src/components/Checkbox/__tests__/Checkbox.test.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import {render, screen} from '@testing-library/react';
+import {Checkbox, CheckboxSize} from '../Checkbox';
+
+const qaId = 'checkbox-component';
+
+describe('Checkbox', () => {
+    test('render checkbox by default', () => {
+        render(<Checkbox />);
+        const checkbox = screen.getByRole('checkbox');
+
+        expect(checkbox).toBeVisible();
+        expect(checkbox).not.toBeDisabled();
+        expect(checkbox).not.toBeChecked();
+    });
+
+    test.each(new Array<CheckboxSize>('m', 'l'))('render with given "%s" size', (size) => {
+        render(<Checkbox size={size} qa={qaId} />);
+        const component = screen.getByTestId(qaId);
+
+        expect(component).toHaveClass(`yc-checkbox_size_${size}`);
```

Comment:
I'm not sure that we need to test component class name here, it is more appropriate for screenshot testing

Reply:
May be we can assert height then?

Reply:
The height has more valid reasons to be changed under the same `m` or `l` classes.

The certain `height` value checks is much more for the screenshots testing. Same to the other CSS behaviour.